### PR TITLE
Issue 146

### PR DESCRIPTION
**Pull request by raymondfeng**
The PR fixes an issue where wsdl `types` element contains multiple schemas with the same or no target namespace.

- connect to #146 

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1 +1,12 @@
-{ "presets": ["es2015"] }
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "node": "4.0.0"
```
Comment (b-admike) [inline .babelrc]: It would be great to know the reason behind this change

Comment (raymondfeng) [reply .babelrc]: It removes the usage of deprecated `babel-preset-es2015` and switches to `babel-preset-env`, which requires a different config.

File: test/wsdl-parse-test.js [hunk 2]
```diff
@@ -60,4 +60,16 @@ describe(__filename, function() {
       done();
     });
   });
+
+  it('should parse types with same or no target namespaces', function(done) {
```
Comment (b-admike) [inline test/wsdl-parse-test.js]: is this test for parsing types with `same target namespaces` or `no target namespaces` or both?

File: src/parser/wsdl/types.js [hunk 3]
```diff
@@ -1,6 +1,5 @@
 'use strict';
 
-var g = require('../../globalize');
```
Comment (b-admike) [inline src/parser/wsdl/types.js]: unused import?

Comment (raymondfeng) [reply src/parser/wsdl/types.js]: No more needed.

**Pull request merged**